Caboodle/FileSystem/FileSystem.uwp.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.ApplicationModel;
+using Windows.Storage;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class FileSystem
+    {
+        private static string cache;
+        private static string appData;
+        private static string userData;
+
+        public static string CacheDirectory
+            => cache ?? (cache = ApplicationData.Current.LocalCacheFolder.Path);
+
+        public static string AppDataDirectory
```

Comment:
Should one of these be roaming? https://docs.microsoft.com/en-us/uwp/api/windows.storage.applicationdata#Examples

Reply:
So I think that "AppData" should just be "LocalFolder" as that is backed up which is the same as the other apps.